Fixed recent error: 'Error authentication, authentication fails 401'

TrianguloY opened the pull request:
Changes: Https is now required, and two additional headers are needed with a valid app id and app key (can be obtained from Inoreader->Settings->Developer).
Tested for personal use with personal keys, working.

Not sure if this project is still active, but anyway it could be useful for other users.

TrianguloY committed:
Fixed recent error: 'Error authentication, authentication fails 401'

Changes: Https is now required, and two additional headers are needed with a valid app id and app key (can be obtained from Inoreader->Settings->Developer).
Tested for personal use with personal keys, working.

fraight1 commented:
Thanks! But how to use it? Sorry i'm an amateur, but I need this extension so bad

TrianguloY commented:
Well, you can import the project and compile it, but of course this requires advanced knowledge with Android Studio and app development.

Let me try to create and publish it directly for an 'install and use' alternative (hopefully in a few days at most).

bartolinio commented:
in that case I am also waiting for it ;)

fraight1 commented:
Happy to hear that, Thank you very very much. You are the man!

TrianguloY committed:
Added apk with fix change.

The only other change was that the actionbarsherlock library was removed due to incompatibilities, an alternative can be considered if necessary (the only change is that now there is no action bar present, but wasn't used anyway).

If you want to use it remember to uninstall first the original one. Keep in mind that future updates of the original app won't be detected by play store, a manual update will be required (if possible I'll notify it from this github).

TrianguloY commented:
Ok, this should do.

- Download from here: https://github.com/TrianguloY/newsplus/blob/master/apk/InoReaderExtension_101_fix.apk (and press download)
- Uninstall existing extension (the extension, not the news+ app. If you checked the 'don't show on launcher' you will need to search it on the android settings).
- Install this one, you need to enable the 'install from external sources' or similar, as with any external apk (installing manual apks can be a risk, I promise this apk is exactly what is expected without any other modification*).
- From the news+ app click on the inoreader option and log in again.

Keep in mind that future updates of the original app won't be detected by play store, a manual update will be required (if possible I'll notify it from this github).

If you find any other problem tell me and I'll try to fix it. Please remember this is a temporary solution and not an official one.

*The actionbarsherlock library was removed due to incompatibilities, this means that the action bar won't be shown (the top bar with the name of the app) but it wasn't used anyway.
